File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Coupons/CouponBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Coupons;
using Ohtap.ECommerce.Domain.Models.Objects.Discounts;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Coupons
{
    public class CouponBasic : ICouponData
    {
        public string ID { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public int Order { get; set; }
        public string Description { get; set; }
        public int State { get; set; }
        public IDiscountData Discount { get; set; }
        public long StartDate { get; set; }
        public long EndDate { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Coupons/CouponBasic.php
<?php


namespace Ohtap\ECommerce\DomainModels\Objects\Coupons;


class CouponBasic implements ICouponData
{
    public $ID;
    public $Discounts;//DiscountBasic
    public $EndDate;
    public $StartDate;
    public $Title;
    public $Name;
    public $State;
    public $OrderNo;
    public $Description;


}